composer.json

```diff
@@ -44,7 +44,8 @@
         "symfony/polyfill-php80": "^1.32",
         "symfony/polyfill-php81": "^1.32",
         "symfony/process": "^5.4.47 || ^6.4.20 || ^7.2",
-        "symfony/stopwatch": "^5.4.45 || ^6.4.19 || ^7.0"
+        "symfony/stopwatch": "^5.4.45 || ^6.4.19 || ^7.0",
+        "symfony/string": "^5.4.45 || ^6.4.19 || ^7.0"
```

Comment:
Note that `symfony/string` is already a non-dev dependency because `symfony/console` depends on `symfony/string`.